homework_11.py

```diff
@@ -0,0 +1,179 @@
+"""This module is a homework 11 Decorators"""
+
+# Положительные аргументы функции
+# Напишите декоратор @validate_arguments, который проверяет, что все аргументы
+# функции являются положительными числами. Если встречается аргумент, не
+# соответствующий этому условию, функция должна вывести сообщение об ошибке.
+
+# Вот некоторые подсказки:
+# Внутри декоратора, используйте цикл for для перебора аргументов функции.
+# Используйте оператор if для проверки, является ли аргумент положительным
+# числом.
+# Если аргумент не соответствует условию, используйте оператор raise для
+# вызова исключения ValueError.
+
+
+def positive_arguments(function):
+    """This decorator function return ValueError if function argument is
+       not a positive number
+
+    function: function to check arguments of which
+
+    return: function that check arguments for type and positivity
+    """
+    def check_args(*args):
+        for arg in args:
+            if not isinstance(arg, (int, float)) or arg <= 0:
+                raise ValueError(f'{arg} is not a positive number')
+        return function
```

Comment:
You are not calling `example_function` function, what you are doing here is just return function object
```suggestion
        return function(*args)
```

Reply:
The same problem for task#2, task#3

Reply:
fixed